src/pygats/recog.py

```diff
@@ -428,3 +428,30 @@ def find_regexp_text(recognized_list: list, pattern):
         if len(match) > 0:
             result.append((roi, content, tuple(match)))
     return list(set(result))
+
+
+def contrast(img):
```

Comment:
What type of img? Is it string or PIL.Image? It should be PIL.Image. So no need to use Image.open(img)...